src/screens/Profile.jsx

```diff
@@ -1,39 +1,57 @@
 import React from "react";
-
+import { useState, useEffect } from 'react';
 import Card from "../components/Card";
+import Loader from "../components/Loader";
 import styles from "./Profile.module.css";
-import { labs } from "../mocks";
 import Tags from "../components/Tags";
+import getDataFromApi from '../utils';
 
 const Profile = (props) => {
-  const lab = labs.find((lab) => lab.id === parseInt(props.match.params.id));
 
-  return (
-    <>
-      <div className={styles.profile}>
-        <div className={styles.hero}>
-          <Card imgUrl={lab.url}></Card>
-        </div>
+  // API URL && Initial State
+  const labsUrl = "https://match-labs-api.herokuapp.com/api/no_auth/labs";
+  const [profile, updateProfile] = useState({});
+  const [loading, updateLoading] = useState(true);
 
-        <div className={styles.rightSide}>
-          <h3 className={styles.name}>{lab.name}</h3>
+  // Request API Data for Lab by ID
+  useEffect(() => {
+    const stopLoading = getDataFromApi(labsUrl, updateProfile, (data) => {
```

Comment:
The show endpoint (`/labs/:id`) might return more information than presented on the index endpoint (`/labs`). When displaying information about a specific resource, you should use the former to ensure all data is fetched!